geode-assembly/build.gradle

```diff
@@ -577,6 +579,9 @@ distributions {
         from {project(':geode-core').tasks.named('raJar')}
         from {project(':geode-core').tasks.named('jcaJar')}
 
+        from project(':geode-modules').tasks.named('module1Jar')
+        from project(':geode-modules').tasks.named('module2Jar')
```

Comment:
I don't think it should exposed outside of the original module

Reply:
Removed.